src/corelibs/U2Core/src/gobjects/PhyTreeObject.cpp

```diff
@@ -67,9 +67,9 @@ PhyTreeObject *PhyTreeObject::createInstance(const PhyTree &tree, const QString
 }
 
 void PhyTreeObject::commit(const PhyTree &tree, const U2EntityRef &treeRef, U2OpStatus &os) {
-    CHECK_EXT(nullptr != tree.data(), os.setError("NULL tree data"), );
-    QByteArray data = NewickPhyTreeSerializer::serialize(tree);
-    RawDataUdrSchema::writeContent(data, treeRef, os);
+    CHECK_EXT(tree.data() != nullptr, os.setError("NULL tree data"), );
+    QString data = NewickPhyTreeSerializer::serialize(tree);
+    RawDataUdrSchema::writeContent(data.toLocal8Bit(), treeRef, os);
```

Comment:
`NewickPhyTreeSerializer::serialize` returns `QByteArray` there's an implicit conversation to `QString` now, and then then you convert `QString` to `QByteArray` again explicitly (`data.toLocal8Bit()`). Replace it with the previous version

Reply:
Didn't you notice this part of the patch: 
+ QString NewickPhyTreeSerializer::serialize(const PhyTree &tree)

I changed the tree serializer to produce a text but not QByteArray